src/components/axes/timeAxis.ts

```diff
@@ -1,6 +1,11 @@
 ///<reference path="../../reference.ts" />
 
 module Plottable {
+
+export enum TimeInterval {
+  second, minute, hour, day, week, month, year
```

Comment:
Is there a problem with it not being an actual `enum`? The type information will still check out for Typescript consumers, and for Javascript consumers it's about the same, except with readable `string` values.